src/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+};
+
+enum Filters {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
```

Comment:
Why do you duplicate enum everywhere?
Move this enum to the separate file and use it everywhere
<img width="543" alt="image" src="https://github.com/user-attachments/assets/119be24a-0d31-4ad1-8380-7af874383b1d">